dsl/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "iroha_dsl"
+version = "2.0.0-pre-rc.16"
+edition = "2021"
+authors = ["Yasser"]
+autobins = false
+
+[lib]
+proc-macro = true
+path = "src/lib.rs"
+
+[dependencies]
+quote = "1.0"
+proc-macro2 = "1.0"
+litrs = "0.4.0"
```

Comment:
A more conventional choice would be to just use `syn::Lit` -- and as @DCNick3 mentioned, you'd want some of its other capabilities sooner or later. E.g. I feel `syn::visit` module could especially come in handy when the tree walks get more elaborate.

Reply:
Created #3756